website/layouts/devices/single.html

```diff
@@ -0,0 +1,79 @@
+{{ define "main" }}
+{{ .Scratch.Set "jsDevice" "http://localhost:9100/device-repository/js/device_single.js" }}
+{{ if not .Site.IsServer }}
+{{ .Scratch.Set "jsDevice" (index .Site.Data.manifest "device_single.js") }}
+{{ end }}
```

Comment:
What is happening here?

Reply:
webpack-dev-server host the assets from http://localhost:9100, this is only necessary from localhost. On production we will used the yarn build output which will be a hashed filename so it looks up the hashed file name in `.Site.Data.manifest`

Run `yarn build` in `/website` to see the outputted manifest.json file `website/data/manifest.json`

Reply:
The webpack outputs are also split up for specific pages so that we don't load all the dependancies for `react` for example in places its not needed

Reply:
And we can't load the local dev environment in `.Site.Data` so that we don't get local development things in our version control?